src/azure_devtools/scenario_tests/utilities.py

```diff
@@ -34,3 +34,18 @@ def get_sha1_hash(file_path):
             sha1.update(data)
 
     return sha1.hexdigest()
+
+
+def is_text_payload(entity):
+    text_content_list = ['application/json', 'application/xml', 'text/html']
```

Comment:
We should put `text/` and use `startswith` in the test at the end of the method. Since all content-type that starts with `text/` are text payload (per HTTP spec)